lib/modules/datasource/puppet-forge/index.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { getPkgReleases } from '..';
+import * as httpMock from '../../../../test/http-mock';
+import { loadFixture } from '../../../../test/util';
+import { PuppetForgeDatasource } from './index';
```

Comment:
```suggestion
import { PuppetForgeDatasource } from '.';
```

Reply:
done

Reply:
👉 https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting